spec/models/export/activity_forecast_columns_spec.rb

```diff
@@ -3,60 +3,195 @@
     DatabaseCleaner.strategy = :transaction
     DatabaseCleaner.start
     @activity = create(:project_activity)
-    @organisation = create(:delivery_partner_organisation, beis_organisation_reference: "BC")
-    @source_fund = Fund.new(1)
     other_activities = create_list(:project_activity, 4)
 
     @activities = [@activity] + other_activities
 
-    create_old_report_and_forecasts
+    q1_2018_report = create(
+      :report,
+      :approved,
+      organisation: @activity.organisation,
+      fund: @activity.associated_fund,
+      financial_quarter: 1,
+      financial_year: 2018
+    )
+    forecasts_for_report_from_table(q1_2018_report,
+      <<~TABLE
+        |financial_quarter|financial_year|value|
+        |4                |2018          | 5000|
+        |1                |2020          |10000|
+        |4                |2020          | 5000|
+        |1                |2021          |40000|
+        |4                |2021          |20000|
+      TABLE
+    )
+
+    q1_2019_report = create(
+      :report,
+      :approved,
+      organisation: @activity.organisation,
+      fund: @activity.associated_fund,
+      financial_quarter: 1,
+      financial_year: 2019
+    )
+    forecasts_for_report_from_table(q1_2019_report,
+      <<~TABLE
+        |financial_quarter|financial_year|value|
+        |1                |2020          | 5000|
+        |4                |2020          | 2500|
+        |1                |2021          |20000|
+        |4                |2021          |10000|
+      TABLE
+    )
```

Comment:
this is a nice way of presenting the forecast fixtures: one table per report.

Reply:
I have you to thank for it! 👍